addons/sourcemod/scripting/vsh/abilities/ability_rage_ghost.sp

```diff
@@ -279,44 +284,29 @@ public void RageGhost_OnThink(SaxtonHaleBase boss)
 			//Other random effects
 			for (int i = 0; i < iLength; i++)
 			{
-				bool bEffectDone = false;
-				
 				//Attempt use random slot
 				if (GetRandomInt(0, 1))
```

Comment:
Do we want to remove this chance? Since this is here to roll for either weapon switch or random angles. Could increase cooldown in trade with it.

Reply:
Yeah I misunderstood this, by glancing over I thought it was just a random chance to trigger either effect at all. Removed it and increased the effect cooldown to 2.5 seconds so it happens 4 total times (as it does that at the beginning of the rage as well).